pkg/sdk/go/channels_test.go

```diff
@@ -4,159 +4,238 @@
 package sdk_test
 
 import (
-	"context"
 	"fmt"
 	"net/http"
 	"net/http/httptest"
 	"reflect"
+	"strings"
```

Comment:
Add test for `ListChannelUsers`

Reply:
Added to users tests. Will be refactored and relocated by #2195